woodwork/tests/schema/test_schema_column.py

```diff
@@ -109,6 +107,32 @@ def test_get_column_dict_params():
     assert column.get('metadata') == {'created_by': 'user1'}
 
 
+def test_get_column_dict_null_params():
+    empty_col = {
+        'logical_type': None,
+        'semantic_tags': set(),
+        'description': None,
+        'metadata': {}
+    }
+    assert _get_column_dict() == empty_col
+
+    just_tags = _get_column_dict(semantic_tags={'numeric', 'time_index'})
+    assert just_tags.get('logical_type') is None
+    assert just_tags.get('semantic_tags') == {'numeric', 'time_index'}
+
+    just_tags = _get_column_dict(logical_type=Integer)
```

Comment:
should this be called `just_ltype` instead?

Reply:
yep, changing